packages/types-algosdk/index.d.ts

```diff
@@ -20,6 +20,42 @@ declare module 'algosdk' {
     statusAfterBlock(lastround: number): Action<any>
   }
 
+ class Kmd {
```

Comment:
please call this class `KMD`

Reply:
ok

Reply:
cannot change that as it is in algosdk module